emanate/__init__.py

```diff
@@ -134,12 +78,50 @@ def del_symlink(self, path_obj):
         return not dest_file.exists()
 
     def run(self):
-        all_files = Path(".").glob("**/*")
+        all_files = Path(self.config.source).glob("**/*")
         files = filter(self.valid_file, all_files)
         list(map(self.function, files))
 
+
+def parse_args(args=None):
```

Comment:
since we're already splitting this function out of the main class, do you think it should be pulled into it's own file? (I'm unsure whether it makes sense to do that or not.)

Reply:
I think it belongs with `main()`; perhaps those two should go into a `main.py`?

Reply:
I _think_ if you put it in `__main__.py` instead of `main.py`, it'll be following typical Python idioms. will need to double-check that, though.

@astronouth7303 am i correct here?

Reply:
I tend to keep `__main__.py` a minimal trampoline into an actual module, but that's kind of a personal preference.

I would encourage the use of a module, though, whether it's main.py or cli.py or __main__.py.

Reply:
`__main__.py` is loaded in an odd context, so I tend to not put too much code in there.

Reply:
alright, in that case let's split it into `cli.py`. /cc @nbraud

Reply:
@duckinator Can you do the split to `cli.py`?  I'm about to go AFK.
Feel free to merge after that if you feel it's good.

Reply:
Yay!  Thanks <3